keys-in.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @module 101/keys-in
+ */
+
+/**
+ * Get all the keys compositing an object, including the `Object.prototype`
+ * @function module:101/keys-in
+ *
+ * @param {object} object - the object from which to extract the keys
+ * @return {array} array of keys
+ */
+module.exports = keysIn;
+
+function keysIn(object) {
+  if (!object) { return []; }
+
+  var keys = [];
+
+  if (object.prototype) { object = Object.create(object.prototype); }
```

Comment:
I think I didn't provide enough details to what I was trying to explain.
In the above example, I was pointing out in the comment that this does *not* have a prototype.
So I am still unsure of what functionality line 19 provides.